garage/tf/algos/npo.py

```diff
@@ -384,14 +387,9 @@ def _build_policy_loss(self, i):
 
             return loss, pol_mean_kl
 
-    def _build_entropy_term(self, i):
+    def _build_entropy_term(self, i, policy_dist_info):
```

Comment:
please don't make these functions dependent on anything but `i`

Reply:
it is very easy to make a loss function which is a tangle of calls to other functions. we instead make sure all components of the loss function are only dependent on `i` (the input spec, basically) which makes it easy to follow.

this will be much easier once the loss function is a model.

Reply:
I see. I modified that since `_build_entropy_term` actually reuses the same `policy_dist_info` symbolic tensor with `_build_policy_loss`. I will come up with another way.

Reply:
you can just not reuse it...

Reply:
We have to reuse it if we don't do dist info sym with a different name. Because each network name is unique. That's why I add "_entropy" after that.

Reply:
We can just append a "_2" to the name in the second call of dist info sym.

Reply:
sure